src/containers/AppStore/containers/AppDetails/AppDetailsTopSection/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {FC, useState} from 'react';
+
+import {Icon, IconType} from '@thenewboston/ui';
+import {useHistory} from 'react-router';
+
+import * as S from './Styles';
+
+type Props = {
+  className?: string;
+  description: string;
+  logoUrl: string;
+  title: string;
+  websiteUrl: string;
+};
+
+const AppDetailsTopSection: FC<Props> = ({className, description, logoUrl, title, websiteUrl}) => {
+  const history = useHistory();
+  const [isWebsiteHovered, setIsWebsiteHovered] = useState(false);
+  const handleVisitWebsite = (): void => {
+    window.open(websiteUrl, '_blank', 'noopener noreferrer');
+  };
```

Comment:
add line breaks b/w hooks(3rd party and react's own) and functions; increases readability.